force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +468,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        Person olderPerson = new Person();
+        if (person1.getAge() > person2.getAge()) {
+             olderPerson = person1;    
+        } else if (person1.getAge() <= person2.getAge()) {
+            olderPerson = person2;
+        }
+        return olderPerson;
     }
```

Comment:
Good work here.  I would not even use that second else if statement.  When you start writing your tests, each branch will need its own test, and each one is a place where a bug can hide. I try to be ruthless about getting rid of conditional logic in my own code. 
```
 public static Person getOlderPerson(Person person1, Person person2) {
      /*  return person1.getAge() > person2.getAge() ? person1 : person2;*/
        if (person1.getAge() > person2.getAge()) {
            return person1;
        }
        return person2;
    }
 ```